### ISSUE TITLE
attribute: better accessible UX

### ISSUE THREAD

**Issue by davidlj95** [davidlj95/website#112]
When activating VoiceOver, an attribute is read as "Empty group". Given the icon is not readable. When using tab sequence, you get the instruction that more content is available. And then, a "description" appears indicating the attribute tooltip content.

Maybe a better experience would be to use `aria-describedby` on the host element. Given [tooltip pattern is in progress in ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/)
